app/admin/events/assignable/AssignedTo.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Admin\Events\Assignable;
+
+use System\Classes\BaseEvent;
+
+class AssignedTo extends BaseEvent
+{
+    public $model;
+    public $log;
+
+    public function __construct($model, $log)
+    {
+        $this->model = $model;
+        $this->log = $log;
+
+        $this->fireBackwardsCompatibleEvent('admin.assignable.assigned', [$model, $log]);
```

Comment:
So instead of this here, see my other comment on Assignable trait :)

Reply:
Yea it made sense to have a BaseEvent class, but don't worry, leave this one for me 👍🏾